spec/spec_tests/data/uri_options/auth-options.yml

```diff
@@ -1,6 +1,6 @@
 tests:
     -
-        description: "Valid auth options are parsed correctly"
+        description: "Valid auth options are parsed correctly (GSSAPI)"
```

Comment:
Submitted a spec ticket to add this to specifications repo: [https://jira.mongodb.org/browse/SPEC-1423](https://jira.mongodb.org/browse/SPEC-1423)